mod-source-record-manager-server/src/main/java/org/folio/verticle/RawMarcChunkConsumersVerticle.java

```diff
@@ -35,4 +37,14 @@ public ProcessRecordErrorHandler<String, String> getErrorHandler() {
     return this.errorHandler;
   }
 
+  @Override
+  public BackPressureGauge<Integer, Integer, Integer> getBackPressureGauge() {
+    /*
+     * The simplest implementation to turn of handling of consumer's load by folio-kafka-wrapper.
```

Comment:
I dont think I am understanding the entire comment after re-reading. I am not sure if the simple implementation is used instead of flow control.

Reply:
Here is this simplest implementation to handle load of kafka consumers inf folio-kafka-wrapper:
https://github.com/folio-org/folio-kafka-wrapper/blob/master/src/main/java/org/folio/kafka/KafkaConsumerWrapper.java#L82
All our consumers use it in order to avoid big pressure to database, oom issues etc. 
For RawMarcChunk consumers we use another flow control mechanism introduced in current PR, so we need to disable this standard one from folio-kafka-wrapper(because if dont need to expect for example to resume consumer in folio-kafka-wrapper if we paused in our flow control service)

Reply:
Link where pause() used in folio-kafka-wrapper: https://github.com/folio-org/folio-kafka-wrapper/blob/master/src/main/java/org/folio/kafka/KafkaConsumerWrapper.java#L197
Link where resume used in folio-kafka-wrapper: https://github.com/folio-org/folio-kafka-wrapper/blob/master/src/main/java/org/folio/kafka/KafkaConsumerWrapper.java#L251

Reply:
thanks for the explanation and references! it makes sense now.
If I may re-write the comment

> Disable back pressure gauge defined by folio-kafka-wrapper by setting this simple implementation. This implementation will not allow folio-kafka-wrapper to pause/resume the topic. Flow control mechanism, defined in this codebase, will be used instead to handle load from DI_RAW_RECORDS_CHUNK_READ topic. Flow control will have exclusive rights to pause/resume the topic.
@see RawRecordsFlowControlService

Reply:
Committed change, thanks